### PR TITLE
Feature/ch1731/create a video section with embedded video

### PR DESCRIPTION
**Pull request by SamMed1**

### CODE REVIEW

**Comment by lewisnyman** [comment]
Looks great! This is how the new heading style looks on Edge:
image

**Comment by lewisnyman** [comment]
Great it looks like this now in Edge 
image

**Comment by SamMed1** [comment]
Great, im happy with that if you are.

File: _includes/components/head.njk [hunk 1]
```diff
@@ -24,7 +24,7 @@
   {% endset %}
   <style>{{ css | cssmin | safe }}</style>
   {% set js %}
-    {# {% include "assets/js/jquery-3.4.1.min.js" %} #}
+    {% include "assets/js/jquery-3.4.1.min.js" %}
```
Comment (lewisnyman) [inline _includes/components/head.njk]: Do we not need jquery anymore? 🤘

Comment (lewisnyman) [reply _includes/components/head.njk]: If so, we can just delete this line

File: pages/cookies.md [hunk 2]
```diff
@@ -43,4 +44,12 @@ This helps us identify how you use this site so we can make it better.
...
-Google isn’t allowed to use or share our analytics data.
\ No newline at end of file
+Google isn’t allowed to use or share our analytics data.
+
+### Play video content (Vimeo)
+Vimeo’s embeddable video player uses essential cookies.
+
+The following cookies are used:
+
+* **player 1 year**
```
Comment (lewisnyman) [inline pages/cookies.md]: Can we follow the same format for the Google Analytics cookies above?

File: _includes/components/head.njk (hunk 1 above)
Comment (SamMed1) [reply _includes/components/head.njk]: This is outdated ^^^ please see recent commit. We do need jquery.

File: pages/cookies.md (hunk 2 above)
Comment (lewisnyman) [reply pages/cookies.md]: This is what they look like in the deploy preview 
image

Comment (SamMed1) [reply pages/cookies.md]: fixed 👍

**Pull request merged**